contracts/EscrowAndStaking/StakingThales.sol

```diff
@@ -292,30 +332,80 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
         }
     }
 
+    /// @notice Get the amount of SNX staked for the account
+    /// @param account to get the amount of SNX staked for
+    /// @return the amount of SNX staked for the account
     function getSNXStaked(address account) external view returns (uint) {
         return _getSNXStakedForAccount(account);
     }
 
-    function getBaseReward(address account) public view returns (uint) {
+    /// @notice Get the base reward amount available for the claim for the account
+    /// @param account to get the base reward amount available for the claim for
+    /// @return the base reward amount available for the claim for the account
+    function getBaseReward(address account) public view returns (uint _baseRewards) {
         if (
-            (_lastStakingPeriod[account] == periodsOfStaking) ||
-            (_stakedBalances[account] == 0) ||
-            (_lastRewardsClaimedPeriod[account] == periodsOfStaking) ||
-            (totalStakedLastPeriodEnd == 0)
+            !((_lastStakingPeriod[account] == periodsOfStaking) ||
```

Comment:
does it mean we want to return always baseRewards?
we should delete the complete `if()` statement

Reply:
It is the same logic as before, just written differently.

Reply:
yea, got it now (didn't notice well the return variable). all good